server/routerlicious/.changeset/silly-papayas-hammer.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/server-routerlicious-base": major
+---
+
+Added support for Redis graceful shutdown
+
+This PR adds a way to ensure that Redis connections are gracefully shut down when disposing service factory resources. There is a new required param, `redisClientConnectionManagers`, in the Nexus, Alfred, and Riddler RunnerFactories. This is scoped to r11s-base.
```

Comment:
Similar to the comment above, let's think of the changesets as release notes (that's exactly how they work in the client release group), so instead of writing them "locally for ourselves", let's write them thinking that the readers are external developers who consume the packages.
```suggestion
Now there's a way to ensure that Redis connections are gracefully shut down when disposing service factory resources. There is a new required param, `redisClientConnectionManagers`, in the Nexus, Alfred, and Riddler RunnerFactories. This is scoped to r11s-base.
```